dashboard/app/models/pd/facilitator_program_registration.rb

```diff
@@ -0,0 +1,286 @@
+# == Schema Information
+#
+# Table name: pd_facilitator_program_registrations
+#
+#  id         :integer          not null, primary key
+#  user_id    :integer          not null
+#  form_data  :text(65535)
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#  teachercon :integer
+#
+# Indexes
+#
+#  index_pd_fac_prog_reg_on_user_id_and_teachercon        (user_id,teachercon) UNIQUE
+#  index_pd_facilitator_program_registrations_on_user_id  (user_id)
+#
+
+require 'state_abbr'
+
+class Pd::FacilitatorProgramRegistration < ActiveRecord::Base
+  LOCATIONS = [
+    'Houston',
+    'Phoenix',
+    'Philadelphia'
+  ].freeze
+
+  DATES = [
+    'June 18 - 23',
+    'July 16 - 21',
+    'July 30 - August 4'
+  ].freeze
+
+  REQUIRED_FIELDS = [
+    :address_street,
+    :address_city,
+    :address_state,
+    :address_zip,
+    :contact_name,
+    :contact_relationship,
+    :contact_phone,
+    :dietary_needs,
+    :live_far_away,
+    :how_traveling,
+    :need_hotel,
+    :need_ada,
+    :liability_waiver,
+    :gender,
+    :race,
+    :age,
+    :grades_taught,
+    :grades_planning_to_teach,
+    :subjects_taught,
+  ].freeze
+
+  OTHER = 'Other'.freeze
+
+  TEACHERCON_DECLINE = 'No - I\'m no longer interested'.freeze
+  TEACHERCON_ALTERNATE = 'No - but I need to attend a different date.'.freeze
+  TRAINING_DECLINE = 'No'.freeze
+  TRAINING_ALTERNATE = 'I want to participate in the program, but I\'m no longer able to attend these dates.'.freeze
+  TRAINING_ALTERNATE_DECLINE = 'I am no longer interested in the Code.org Facilitator Development Program.'.freeze
+
+  OPTIONS = {
+    confirm_teachercon_date: [
+      'Yes',
+      TEACHERCON_ALTERNATE,
+      TEACHERCON_DECLINE
+    ],
+
+    alternate_teachercon_date: [
+      "TeacherCon 1: #{DATES[0]}",
+      "TeacherCon 2: #{DATES[1]}",
+      "TeacherCon 3: #{DATES[2]}"
+    ],
+
+    confirm_training_date: [
+      'Yes',
+      TRAINING_DECLINE
+    ],
+
+    decline_training_date: [
+      TRAINING_ALTERNATE,
+      TRAINING_ALTERNATE_DECLINE
+    ],
+
+    csd_alternate_training_date: [
+      'July 22 - 23 (immediately following TeacherCon 2)',
+      'August 5 - 6 (immediately following TeacherCon 3)'
+    ],
+
+    csp_alternate_training_date: [
+      'June 24 - 25 (immediately following TeacherCon 1)',
+      'July 22 - 23 (immediately following TeacherCon 2)',
+      'August 5 - 6 (immediately following TeacherCon 3)'
+    ],
+
+    address_state: STATE_ABBR_WITH_DC_HASH.stringify_keys,
+
+    dietary_needs: [
+      'None',
+      'Vegetarian',
+      'Gluten Free',
+      'Food allergy',
+      OTHER
+    ],
+
+    live_far_away: [
+      'Yes',
+      'No'
+    ],
+
+    how_traveling: [
+      'Driving',
```

Comment:
checked with Malorie, and she'd prefer we keep this set of options the same as in the teacher application.